index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">  
+  <header>
+    <div class="top-header"><img src="images/Time.png" alt="time">
+    <img src="images/wifi.png" alt="wifi">
+  </div>
+    <div class="bottom-header">
+      <h1>Welcome</h1>
+      <span class="identical-to">&#8801;</span>
+  </div>
+  </header> 
+  <main>
+    <div class="intro">
+    <h2 class="first-name">Hey there. I'm Stephen</h2>
+    <h2 class="first-name software-dev">I'm a software developer</h2>
+  </div>
+    <p class="paragraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.</p>    
+    <div class="social">
+    <img class="social-img" src="images/twitter.png" alt="twitter">
+    <img class="social-img" src="images/Linkedin.png" alt="linkedin">
+    <img class="social-img" src="images/medium.png" alt="gmail">
+    <img class="social-img" src="images/github.png" alt="github">
+    <img class="social-img" src="images/angellist.png" alt="angellist">    
+  </div>
```

Comment:
- [ ] Great work adding the bottom icons, however, it will be good practice to use a `ul` tag to group the icons and wrap each icon in a `li` tag for semantic purposes. You can refer to the link provided in one of the above comments for semantic tags.

Reply:
Awesome, resolved!